dashboard/modules/job/job_manager.py

```diff
@@ -447,7 +447,16 @@ async def _monitor_job(
 
         while is_alive:
             try:
-                await job_supervisor.ping.remote()
+                # Workaround for https://github.com/ray-project/ray/issues/28058.
+                # Once fixed, replace with `await job_supervisor.ping.remote()`.
+                ref = job_supervisor.ping.remote()
+                not_ready = [ref]
+                while not_ready:
+                    ready, not_ready = ray.wait(ref, timeout=0.1)
```

Comment:
```suggestion
                    ready, not_ready = ray.wait(not_ready, timeout=0.1)
```

I believe `ray.wait()` expects a list of references.